packages/examples/src/day6.ts

```diff
@@ -0,0 +1,141 @@
+import { registerExamples } from './register';
```

Comment:
Do we have day6 twice now? If yes, why so?

Reply:
something went wrong, i will rebase this and clean up